src/helpers/generateToken.js

```diff
@@ -0,0 +1,18 @@
+import jwt from 'jsonwebtoken';
+import dotenv from 'dotenv';
```

Comment:
I think this doesn't align with the reset password token and reset time column we discussed when creating the user tables. The token should be generated with crypto(node) instead of JWT since we don't need a payload object and saved in the database with an expiring time.

Reply:
I think there is no need for hitting the database when a token that has an expiring time with the user's id can be generated to check for the authenticity of the user that received the email because I feel the same thing we want to achieve can be achieved without having to send a server request.